roles/install_nextcloud/defaults/main.yml

```diff
@@ -32,6 +32,7 @@ php_pkg_spe:
   - "php{{ php_ver }}-mbstring"
   - "php{{ php_ver }}-xml"
   - "php{{ php_ver }}-zip"
+  - "php{{ php_ver }}-apcu"
```

Comment:
This is already installed in [php_install.yml](https://github.com/nextcloud/ansible-collection-nextcloud-admin/blob/8f5d564bcedc603205d4346db21388a390689419/roles/install_nextcloud/tasks/php_install.yml#L42)

Reply:
OK, interesting. I had some issue with Debian/Ubuntu fresh-installs and only some packages selected for pre-installation. Would you want me to remove this, it otherwise shouldn't hurt.

Reply:
Yes please.

Reply:
Done - at least for me it looks like this. In case not, this is attributable to my insufficient git-knowledge.